### PR TITLE
Add support for secure data in pushover options.

### PR DESCRIPTION
**Pull request by ludovicchabant**

### CODE REVIEW

**Comment by stfnm** [comment]
Thanks! I'll have a look at it soon.

File: perl/pushover.pl [hunk 1]
```diff
@@ -78,7 +78,7 @@ sub init_config
 			weechat::config_set_plugin($option, $OPTIONS_DEFAULT{$option}[0]);
 			$OPTIONS{$option} = $OPTIONS_DEFAULT{$option}[0];
 		} else {
-			$OPTIONS{$option} = weechat::config_get_plugin($option);
+			$OPTIONS{$option} = weechat::string_eval_expression(weechat::config_get_plugin($option), {}, {}, {});
```
Comment (stfnm) [inline perl/pushover.pl]: You forgot the same bit for the "config_cb" subroutine, which is called whenever a config value is changed.

However I think it's even better to evaluate the expressions directly in the "notify" subroutine where they're being used, so I did that quickly myself in commit 1cb5b348e934a2665d6a28a79e6c2d28e9ae188a. :)

**Comment by ludovicchabant** [comment]
Good, thanks for the fixes! (it was my first foray into both Perl code and Weechat scripts so it's safe to say I had absolutely no idea what I was doing! :) )
Do you know when it will be pushed to the official Weechat scripts repo?

**Comment by stfnm** [comment]
It shouldn't take long: https://github.com/weechat/scripts/pull/53

**Pull request closed without merging**